runtime/smithy-client/api/smithy-client.api

```diff
@@ -115,24 +155,20 @@ public abstract interface class aws/smithy/kotlin/runtime/client/SdkClient$Build
 
 public abstract interface class aws/smithy/kotlin/runtime/client/SdkClientConfig {
 	public abstract fun getClientName ()Ljava/lang/String;
+	public abstract fun getLogMode ()Laws/smithy/kotlin/runtime/client/LogMode;
 	public abstract fun getRetryPolicy ()Laws/smithy/kotlin/runtime/retries/policy/RetryPolicy;
 	public abstract fun getRetryStrategy ()Laws/smithy/kotlin/runtime/retries/RetryStrategy;
-	public abstract fun getSdkLogMode ()Laws/smithy/kotlin/runtime/client/SdkLogMode;
 }
 
 public abstract interface class aws/smithy/kotlin/runtime/client/SdkClientConfig$Builder : aws/smithy/kotlin/runtime/util/Buildable {
 	public abstract fun getClientName ()Ljava/lang/String;
+	public abstract fun getLogMode ()Laws/smithy/kotlin/runtime/client/LogMode;
 	public abstract fun getRetryPolicy ()Laws/smithy/kotlin/runtime/retries/policy/RetryPolicy;
 	public abstract fun getRetryStrategy ()Laws/smithy/kotlin/runtime/retries/RetryStrategy;
-	public abstract fun getSdkLogMode ()Laws/smithy/kotlin/runtime/client/SdkLogMode;
 	public abstract fun setClientName (Ljava/lang/String;)V
+	public abstract fun setLogMode (Laws/smithy/kotlin/runtime/client/LogMode;)V
 	public abstract fun setRetryPolicy (Laws/smithy/kotlin/runtime/retries/policy/RetryPolicy;)V
 	public abstract fun setRetryStrategy (Laws/smithy/kotlin/runtime/retries/RetryStrategy;)V
-	public abstract fun setSdkLogMode (Laws/smithy/kotlin/runtime/client/SdkLogMode;)V
-}
-
-public final class aws/smithy/kotlin/runtime/client/SdkClientConfig$DefaultImpls {
-	public static fun getSdkLogMode (Laws/smithy/kotlin/runtime/client/SdkClientConfig;)Laws/smithy/kotlin/runtime/client/SdkLogMode;
 }
```

Comment:
**Comment**: TIL `DefaultImpls` is yet another synthetic class the compiler adds.